bokeh/tile_providers.py

```diff
@@ -218,6 +241,10 @@ def get_provider(self, provider_name):
     STAMEN_TONER = Vendors.STAMEN_TONER
     STAMEN_TONER_BACKGROUND = Vendors.STAMEN_TONER_BACKGROUND
     STAMEN_TONER_LABELS = Vendors.STAMEN_TONER_LABELS
+    STAMEN_TONER_LABELS = Vendors.STAMEN_TONER_LABELS
```

Comment:
copy-paste error I think, this line duplicates the line right above

Reply:
> copy-paste error I think, this line duplicates the line right above

yes, ditto - thanks - mistaken duplicate line - corrected